### PR TITLE
Update Token Mapper guide and Polygon Wiki thumbnail

### PR DESCRIPTION
**Pull request by ashucoder9**

### CODE REVIEW

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 1]
```diff
@@ -34,13 +43,38 @@ In  case of a custom child mapping, there is a checklist that you need to finish
 
 **Non Polygon-Mintable**
 
-1. The deposit and withdraw functions are present on the child token contract. (Reference Template contract - [ERC20](https://github.com/maticnetwork/pos-portal/blob/master/flat/ChildERC20.sol#L1492-#L1508), [ERC721](https://github.com/maticnetwork/pos-portal/blob/master/flat/ChildERC721.sol#L2157-#L2238), [ERC1155](https://github.com/maticnetwork/pos-portal/blob/master/flat/ChildERC1155.sol#L1784-#L1818))
-2. Only the ChildChainManagerProxy address has the right to call the deposit function. (ChildChainManagerProxy - on [Mumbai](https://mumbai.polygonscan.com/address/0xb5505a6d998549090530911180f38aC5130101c6/transactions) , on [Polygon Mainnet](https://polygonscan.com/address/0xA6FA4fB5f76172d178d61B04b0ecd319C5d1C0aa/) )
-3. Mint function is an internal function ( This gets called by deposit function internally )
+The Tokens that will be minted on Polygon first and subsequently migrated to Ethereum are called **Mintable Tokens**. Meanwhile, Tokens that already exist on Ethereum but have to be moved to Polygon are referred to as **Non-Mintable Tokens**.
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:46]:
```suggestion
The tokens to be minted on Polygon and subsequently migrated to Ethereum are called **mintable tokens**, whereas tokens that already exist on Ethereum and are migrated over to Polygon are referred to as **non-mintable tokens**.

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 id: submit-mapping-request
-title: Mapping Request
-description:  "Steps to submit a mapping request."
+title: Mapping Tokens
+description:  A guide on how to map tokens between Ethereum and Polygon Chains using the PoS Bridge
 keywords:
   - docs
   - matic
+  - token mapping
+  - PoS Bridge
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:9]:
```suggestion
- pos bridge
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 3]
```diff
@@ -14,7 +17,13 @@ Mapping is necessary in order to transfer your assets to and from the Ethereum a
 
 ### Steps to submit a mapping request
 
-The mapping request has to be submitted on [https://mapper.polygon.technology/](https://mapper.polygon.technology/). You can then click on the "Map New Token" button on the top right corner to create a new mapping request.
+Polygon PoS Bridge is available for both Mainnet as well as Mumbai Testnet. More information about the PoS Bridge can be found [<ins>here</ins>](/docs/develop/ethereum-polygon/pos/getting-started).
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:20]:
```suggestion
The Polygon PoS bridge is available for both mainnet as well as Mumbai testnet. More information about the bridge can be found [<ins>here</ins>](/docs/develop/ethereum-polygon/pos/getting-started).

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 4]
```diff
@@ -14,7 +17,13 @@ Mapping is necessary in order to transfer your assets to and from the Ethereum a
...
+
+:::
+
+## How To Submit A Mapping Request
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:24]:
```suggestion
## How to submit a mapping request
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md (hunk 3 above)
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:20]:
```suggestion
The Polygon PoS bridge is available for both mainnet as well as Mumbai testnet. More information about the bridge can be found [<ins>here</ins>](/docs/develop/ethereum-polygon/pos/getting-started).

File: docs/develop/ethereum-polygon/submit-mapping-request.md (hunk 4 above)
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:24]:
```suggestion
## How to submit a mapping request
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 5]
```diff
@@ -34,13 +43,38 @@ In  case of a custom child mapping, there is a checklist that you need to finish
...
-2. Only the ChildChainManagerProxy address has the right to call the deposit function. (ChildChainManagerProxy - on [Mumbai](https://mumbai.polygonscan.com/address/0xb5505a6d998549090530911180f38aC5130101c6/transactions) , on [Polygon Mainnet](https://polygonscan.com/address/0xA6FA4fB5f76172d178d61B04b0ecd319C5d1C0aa/) )
-3. Mint function is an internal function ( This gets called by deposit function internally )
+The Tokens that will be minted on Polygon first and subsequently migrated to Ethereum are called **Mintable Tokens**. Meanwhile, Tokens that already exist on Ethereum but have to be moved to Polygon are referred to as **Non-Mintable Tokens**.
+
+:::
+
+In case of a custom token mapping, you can visit our [Fx-Portal](/docs/develop/l1-l2-communication/fx-portal) and use the data provided to build your custom token.
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:50]:
```suggestion
In the case where you require custom token mapping, you can visit the [Fx-Portal](/docs/develop/l1-l2-communication/fx-portal) and use the data provided to build your custom token.

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 6]
```diff
@@ -34,13 +43,38 @@ In  case of a custom child mapping, there is a checklist that you need to finish
...
+
+## Mapping Checklist
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:52]:
```suggestion
## Mapping checklist
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 7]
```diff
@@ -34,13 +43,38 @@ In  case of a custom child mapping, there is a checklist that you need to finish
...
+
+### Mintable Tokens
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:54]:
```suggestion
### Mintable tokens
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md (hunk 6 above)
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:52]:
```suggestion
## Mapping checklist
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md (hunk 7 above)
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:54]:
```suggestion
### Mintable tokens
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 8]
```diff
@@ -34,13 +43,38 @@ In  case of a custom child mapping, there is a checklist that you need to finish
...
+
+:::tip
+
+Checkout the documentation for Mintable tokens available at [<ins>https://wiki.polygon.technology/docs/develop/ethereum-polygon/mintable-assets</ins>](/docs/develop/ethereum-polygon/mintable-assets)
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:58]:
```suggestion
Checkout the documentation for mintable tokens available at [<ins>https://wiki.polygon.technology/docs/develop/ethereum-polygon/mintable-assets</ins>](/docs/develop/ethereum-polygon/mintable-assets).
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 9]
```diff
@@ -34,13 +43,38 @@ In  case of a custom child mapping, there is a checklist that you need to finish
...
 3. The root chain contract is a standard [ERC20](https://github.com/maticnetwork/pos-portal/blob/master/flat/DummyMintableERC20.sol#L1481)/[ERC721](https://github.com/maticnetwork/pos-portal/blob/master/flat/DummyMintableERC721.sol#L2169)/[ERC1155](https://github.com/maticnetwork/pos-portal/blob/master/flat/DummyMintableERC1155.sol#L1785)
-4.  The mint function on the root contract can only be called by the corresponding token, PredicateProxyAddress (PredicateProxy addresses for each token type can be found [here](/docs/develop/ethereum-polygon/mintable-assets#contract-to-be-deployed-on-ethereum).
\ No newline at end of file
+4.  The mint function on the root contract can only be called by the corresponding token, **PredicateProxyAddress**. (PredicateProxyAddress for each token type can be found [here](/docs/develop/ethereum-polygon/mintable-assets#contract-to-be-deployed-on-ethereum))
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:65]:
```suggestion
4. The `mint` function on the *root* contract can only be called by the corresponding token, **PredicateProxyAddress**. (PredicateProxyAddress for each token type can be found [here](/docs/develop/ethereum-polygon/mintable-assets#contract-to-be-deployed-on-ethereum)).

File: docs/develop/ethereum-polygon/submit-mapping-request.md (hunk 2 above)
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:9]:
```suggestion
- pos bridge
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md (hunk 9 above)
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:65]:
```suggestion
4. The `mint` function on the *root* contract can only be called by the corresponding token, **PredicateProxyAddress**. (PredicateProxyAddress for each token type can be found [here](/docs/develop/ethereum-polygon/mintable-assets#contract-to-be-deployed-on-ethereum)).
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 10]
```diff
@@ -34,13 +43,38 @@ In  case of a custom child mapping, there is a checklist that you need to finish
...
+
+### Non-Mintable Tokens
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:67]:
```suggestion
### Non-mintable tokens
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 11]
```diff
@@ -34,13 +43,38 @@ In  case of a custom child mapping, there is a checklist that you need to finish
...
+
+1. The deposit and withdraw functions are present in the child token contract. (Reference Template Contract - [ERC20](https://github.com/maticnetwork/pos-portal/blob/master/flat/ChildERC20.sol#L1492-#L1508), [ERC721](https://github.com/maticnetwork/pos-portal/blob/master/flat/ChildERC721.sol#L2157-#L2238), [ERC1155](https://github.com/maticnetwork/pos-portal/blob/master/flat/ChildERC1155.sol#L1784-#L1818))
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:69]: 1. The `deposit` and `withdraw` functions are present in the *child* token contract. (Reference template contract - [ERC20](https://github.com/maticnetwork/pos-portal/blob/master/flat/ChildERC20.sol#L1492-#L1508), [ERC721](https://github.com/maticnetwork/pos-portal/blob/master/flat/ChildERC721.sol#L2157-#L2238), [ERC1155](https://github.com/maticnetwork/pos-portal/blob/master/flat/ChildERC1155.sol#L1784-#L1818)).
```

File: docs/develop/ethereum-polygon/submit-mapping-request.md [hunk 12]
```diff
@@ -34,13 +43,38 @@ In  case of a custom child mapping, there is a checklist that you need to finish
...
+2. Only the **ChildChainManagerProxy** address has the right to call the deposit function. (ChildChainManagerProxy - on [Mumbai](https://mumbai.polygonscan.com/address/0xb5505a6d998549090530911180f38aC5130101c6/transactions) , on [Polygon Mainnet](https://polygonscan.com/address/0xA6FA4fB5f76172d178d61B04b0ecd319C5d1C0aa/))
+3. Mint function is an internal function. (This gets called by deposit function internally)
+
+## Video Guide
```
Comment (DannyS03) [inline docs/develop/ethereum-polygon/submit-mapping-request.md:73]:
```suggestion
## Video guide
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]

**Comment by ashucoder9** [comment]
Added changes @DannyS03

**Review by DannyS03 — APPROVED** [review]

**Comment by DannyS03** [comment]
nice:) feel free to merge when ready @ashucoder9

**Pull request merged**